README.md

```diff
@@ -247,11 +247,156 @@ export default function (eleventyConfig) {
 The `defaultLanguage` can be overridden by passing a new value to the `defaultLanguage` options key when registering
 `eleventy-plugin-fluid`.
 
-By default, `eleventy-plugin-fluid` also configures a [`localesDirectory`](https://github.com/sgissinger/eleventy-plugin-i18n-gettext#localesdirectory)
-for `eleventy-plugin-i18n-getttext` as `./src/_locales`. This can be overridden by passing a new value to the
-`localesDirectory` options key when registering `eleventy-plugin-fluid`.
+The localization shortcodes included in `eleventy-plugin-fluid` require a [global data object](https://www.11ty.dev/docs/data-global/)
+called `translations` with the following shape:
 
-`eleventy-plugin-fluid` also provides two localization-related helpers:
+```json
+{
+    "en": {
+        "hello": "Hello {{name}}!"
+    },
+    "fr": {
+        "hello": "Bonjour {{name}} !"
+    }
+}
+```
+
+`eleventy-plugin-fluid` also provides four localization-related helpers:
+
+#### `__`
+
+[`__`](src/utils/translation.js#L21) is used to translate a string, substituting values for placeholders where
+required. For example:
+
+```js
+import { __ } from "eleventy-plugin-fluid";
+
+console.log(__(
+    "hello",
+    {
+        "name": "world"
+    },
+    {
+        "locale": "fr",
+        "translations": {
+            "en": {
+                "hello": "Hello {{name}}!"
+            },
+            "fr": {
+                "hello": "Bonjour {{name}} !"
+            }
+        }
+    }
+));
+```
+
+Result: `Bonjour world !`
+
+The first parameter is the key for the translation string in the `translations` object, the second parameter is an
+object of values to substitute for placeholders in the translation string, and the third parameter is the data object
+containing the `locale` and `translations` values. If the third parameter is not provided, the function will try to
+retrieve these values from `this.ctx` which provides access to Eleventy's global data.
+
+`eleventy-plugin-fluid` also provides a [shortcode](https://www.11ty.dev/docs/shortcodes/) based on this function:
+
+<details>
+<summary>Liquid</summary>
+
+```liquid
+{% __ 'hello', {name: 'world'} %}
+```
```

Comment:
When I leave the default htmlTemplateEngine configuration this syntax doesn't work. I get the following error:

```log
[11ty] Problem writing Eleventy templates:
[11ty] 1. Having trouble rendering liquid template ./fr/index.html (via TemplateContentRenderError)
[11ty] 2. invalid syntax at line 1 col 10:
[11ty] 
[11ty] 1  'hello', {name: 'world'}
[11ty]             ^, file:./fr/index.html, line:7, col:12 (via RenderError)
[11ty] 3. invalid syntax at line 1 col 10:
[11ty] 
[11ty] 1  'hello', {name: 'world'}
[11ty]             ^
[11ty] 
[11ty] Original error stack trace: Error: invalid syntax at line 1 col 10:
[11ty] 
[11ty] 1  'hello', {name: 'world'}
[11ty]             ^
[11ty]     at Lexer._token (/Users/jo/Desktop/test-11ty-plugin/node_modules/moo/moo.js:559:17)
[11ty]     at Lexer.next (/Users/jo/Desktop/test-11ty-plugin/node_modules/moo/moo.js:506:19)
[11ty]     at Liquid.parseArguments (file:///Users/jo/Desktop/test-11ty-plugin/node_modules/@11ty/eleventy/src/Engines/Liquid.js:164:17)
[11ty]     at Tag.render (file:///Users/jo/Desktop/test-11ty-plugin/node_modules/@11ty/eleventy/src/Engines/Liquid.js:206:28)
[11ty]     at render.next (<anonymous>)
[11ty]     at toPromise (/Users/jo/Desktop/test-11ty-plugin/node_modules/liquidjs/dist/liquid.node.js:524:32)
[11ty]     at toPromise (/Users/jo/Desktop/test-11ty-plugin/node_modules/liquidjs/dist/liquid.node.js:530:25)
[11ty]     at async toPromise (/Users/jo/Desktop/test-11ty-plugin/node_modules/liquidjs/dist/liquid.node.js:532:25)
[11ty]     at async Template._render (file:///Users/jo/Desktop/test-11ty-plugin/node_modules/@11ty/eleventy/src/TemplateContent.js:592:19)
[11ty]     at async TemplateMap.populateContentDataInMap (file:///Users/jo/Desktop/test-11ty-plugin/node_modules/@11ty/eleventy/src/TemplateMap.js:537:7)
[11ty] Wrote 0 files in 0.33 seconds (v3.0.0-alpha.18)
```

The html file is:

```html
<!doctype html>
<html lang="fr">
    <head>
        <title>Titre de la page</title>
    </head>
    <body>
        <p>{% __ 'hello', {name: 'world'} %}</p>
        <p>{% __ 'hello', {name: 'world'}, {locale: page.lang, translations: translations} %}</p>
    </body>
</html>
```

Reply:
Just to refresh my memory on this, @jobara — we determined that we can only pass an object directly with Nunjucks, right?

Reply:
Yes that seemed to be the case.